Detectors/ITSMFT/ITS/simulation/src/Detector.cxx

```diff
@@ -917,8 +917,8 @@ void Detector::constructDetectorGeometry()
 
     LOG(DEBUG1) << "mBuildLevel: " << mBuildLevel[j] << FairLogger::endl;
 
-    if (mStaveThickness[j] != 0) {
-      mGeometry[j]->setStaveThick(mStaveThickness[j]);
+    if (mChipThickness[j] != 0) {
+      mGeometry[j]->setChipThick(mChipThickness[j]);
```

Comment:
Is the `V1Layer` class supposed to work with this as well? Then the function `SetStaveThickness` must be renamed.